scripts/languageserver/parse.jl

```diff
@@ -0,0 +1,243 @@
+type VarInfo
+    t
+    doc::String
+end
+
+type Block
+    uptodate::Bool
+    ex::Any
+    range::Range
+    name::String
+    var::VarInfo
+    localvar::Dict{String,VarInfo}
+    diags::Vector{Diagnostic}
+end
+
+function Block(utd, ex, r::Range)
+    t, name, doc, lvars = classify_expr(ex)
+    ctx = LintContext()
+    ctx.lineabs = r.start.line+1
+    dl = r.end.line-r.start.line-ctx.line
+    Lint.lintexpr(ex, ctx)
+    diags = map(ctx.messages) do l
+        return Diagnostic(Range(Position(r.start.line+l.line+dl-1, 0), Position(r.start.line+l.line+dl-1, 100)),
```

Comment:
The columns here are incorrect. #60 fixes that in the current version, maybe you can just adapt that code here?